vercel.json

```diff
@@ -1,4 +1,15 @@
 {
+  "headers": [
```

Comment:
Are the nested `headers` objects here really correct? Without even looking at the docs, this looks fishy.

Reply:
See https://github.com/dimasahmad/install.python-poetry.org/commit/7a01d87dc60b674e8e9f45c52841a2da92a8535c#commitcomment-62669502 for the resulting site with this configuration.

I had a hard time looking for how to implement the headers within vercel.json, but found the [vercel.json](https://github.com/vercel/vercel/blob/main/vercel.json) in vervel's repo contains the headers config.